torchaudio/functional/functional.py

```diff
@@ -1422,7 +1420,7 @@ def resample(
         lowpass_filter_width: int = 6,
         rolloff: float = 0.99,
         resampling_method: str = "sinc_interpolation",
-        beta: Optional[float] = None,
+        beta: float = 14.769656459379492,
```

Comment:
@mthrok is it okay to use type float instead of Optional[float] even though we don't use beta in the case of resampling_method=sinc_interpolation? I get a jit error because of `beta_tensor = torch.tensor(beta)` when beta is of type `Optional[float]`, not sure if there is a better way to handle this

Reply:
I think you need to perform [optional type refinement](https://pytorch.org/docs/stable/jit_language_reference.html#optional-type-refinement). 

```
beta_ = 14.769656459379492
if beta is not None:
    beta_ = beta
beta_tensor = torch.tensor(beta_)
```

Reply:
or 

```python
if beta is not None:
    beta_tensor = torch.tensor(beta)
else:
    beta_tensor = torch.tensor(14.769656459379492)
```